mern-ecommerce/routes/order.js

```diff
@@ -29,15 +29,23 @@ router.get(
   requireSignin,
   isAuth,
   isAdmin,
-  getStatusValues
+  getStatus
 );
 
 router.put(
   '/order/:orderId/status/:userId',
   requireSignin,
   isAuth,
   isAdmin,
-  updateOrderStatus
+  updateOrder
+);
+
+router.delete(
+  '/order/:orderId/:userId',
```

Comment:
In the route '/order/:orderId/:userId', the parameter names 'orderId' and 'userId' should be consistently used in the corresponding controller function 'remove' to avoid any mismatch or errors during execution.